src/connection.rs

```diff
@@ -375,6 +409,15 @@ impl Connection {
     pub fn set_read_timeout(&self, dur: Option<Duration>) -> RedisResult<()> {
         self.con.borrow().set_read_timeout(dur)
     }
+
+
+    /// Returns the connection status.
+    ///
+    /// The connection is open until any `read_response` call failed to
+    /// parse a command.
```

Comment:
We should make it clear that not the parsing makes the connection be closed, but an actual socket error.

Reply:
I tried to make it clearer.